model/ddl.go

```diff
@@ -125,6 +125,9 @@ type Job struct {
 	// Query string of the ddl job.
 	Query      string       `json:"query"`
 	BinlogInfo *HistoryInfo `json:"binlog"`
+
+	// Version starts at bootstrap version 15.
+	Version int64 `json:"version"`
```

Comment:
The `Version` won't alreays fellow `bootstrap version`, begin from 15 is just for simple.